### ISSUE TITLE
Slovakia VAT rate will change from 20% to 23% on 2025-01-01

### ISSUE THREAD

**Issue by MartinMystikJonas** [driesvints/vat-calculator#185]
What is correct way to cprepare fot this change? Should I for example prepare PR that would be merged on Jan 1st?

**Comment by driesvints** [comment]
Yes send in a PR however I won't be there to merge it as I'm on vacation during that time. I'll merge it as soon as I'm able. Thanks!

**Comment by spaze** [comment]
Hey @driesvints I think I'll be able to merge and tag a release Jan 1 soon after midnight either CET or UTC, up to you. Let me know if you need help releasing the change :-)

**Comment by driesvints** [comment]
Thank you @spaze, I really appreciate that. You only need to make a GitHub release and everything else like the changelog will be done automatically. Make sure to define the new tag without the `v` prefix. Thanks! 🙏

**Comment by spaze** [comment]
Cool @driesvints, thanks for the info.

@MartinMystikJonas can you please prepare the PR? @driesvints can you please then review it so it's ready for me to merge?

**Comment by MartinMystikJonas** [comment]
I will prepere it sometime this week.

**Comment by MartinMystikJonas** [comment]
#186
